develop/tutorials/articles/14-themes-and-layout-templates/02-layout-templates/00-layout-templates-intro.markdown

```diff
@@ -0,0 +1,7 @@
+# Layout Templates [](id=layout-templates-intro)
+
+@product@ includes several default layout templates out-of-the-box that you 
+can use to organize content on your pages. If you require a custom 
+configuration, you can create layout templates to cover your specific needs. In 
+this section of tutorials, you'll learn how to develop layout templates for 
+@product@.
```

Comment:
Hey @mwilliams2014 can you improve this intro so it explains what Layout Templates are and why (i.e., use cases) you'd want to create your own?

Reply:
Sorry this got buried @sez11a. Yes, I'll update it and push the changes.